src/sentry/web/api.py

```diff
@@ -1002,6 +1003,12 @@ def get(self, request, **kwargs):
         return HttpResponse(json.dumps(schemas.EVENT_SCHEMA), content_type='application/json')
 
 
+class ClientConfigView(BaseView):
```

Comment:
Where should this endpoint live?

Reply:
I think this is a reasonable spot. The other views in this module have similar authentication behavior.